siac/consensuscmd_test.go

```diff
@@ -16,28 +16,28 @@ func TestEstimatedHeightAt(t *testing.T) {
 	}{
 		// Test on the same block that is used to estimate the height
 		{
-			time.Date(2016, time.May, 11, 19, 33, 0, 0, time.UTC),
-			5e4,
+			time.Date(2017, time.April, 13, 23, 29, 49, 0, time.UTC),
+			1e5,
 		},
 		// 4 minutes later
 		{
-			time.Date(2016, time.May, 11, 19, 37, 0, 0, time.UTC),
-			5e4,
+			time.Date(2017, time.April, 13, 23, 33, 49, 0, time.UTC),
+			1e5,
```

Comment:
100e3, same for all the others.

Remember that we do our constants in sets of 3 (e3, e6, e9, etc.) because it maps easily to the thousand/million/billion/etc that people are already familiar with.